app/javascript/react/screens/App/Plan/plan.planRequests.fixtures.js

```diff
@@ -248,6 +248,70 @@ for (let i = 0; i < 30; i += 1) {
   sampleTasks.push(copy);
 }
 
+const completeSampleTasksWithFailures = [];
+
+for (let i = 0; i < 30; i += 1) {
+  const copy = JSON.parse(JSON.stringify(sampleTask));
+  copy.id = i;
+  copy.miq_request_id = i;
+  copy.options.delivered_on = moment(copy.options.delivered_on)
+    .subtract(Math.floor(Math.random() * 48), 'hours')
+    .format();
+  copy.options.updated_on = moment(copy.options.delivered_on)
+    .add(Math.floor(Math.random() * 100), 'm')
+    .format();
+  copy.message = 'VM Transformations completed';
+
+  if (i % 5 === 4) {
+    copy.options.progress.current_description =
+      'Virtual machine migration failed.';
+    copy.status = 'Error';
+    for (let j = 0; j < copy.options.virtv2v_disks.length; j += 1) {
+      copy.options.virtv2v_disks[j].percent = Math.floor(Math.random() * 100);
+      copy.options.virtv2v_disks[j].size = Math.floor(
+        Math.random() * 17179869184
+      );
+    }
+  } else {
+    copy.options.progress.current_description = 'Virtual machine migrated';
+    for (let j = 0; j < copy.options.virtv2v_disks.length; j += 1) {
+      copy.options.virtv2v_disks[j].percent = 100;
+      copy.options.virtv2v_disks[j].size = Math.floor(
+        Math.random() * 17179869184
+      );
+    }
+  }
+
+  copy.options.transformation_host_name = `rhvh${i}.example.com`;
+  completeSampleTasksWithFailures.push(copy);
+}
+
+const completeSampleTasks = [];
+
+for (let i = 0; i < 30; i += 1) {
+  const copy = JSON.parse(JSON.stringify(sampleTask));
+  copy.id = i;
+  copy.miq_request_id = i;
+  copy.options.delivered_on = moment(copy.options.delivered_on)
+    .subtract(Math.floor(Math.random() * 48), 'hours')
+    .format();
+  copy.options.updated_on = moment(copy.options.delivered_on)
+    .add(Math.floor(Math.random() * 100), 'm')
+    .format();
+  copy.message = 'VM Transformations completed';
```

Comment:
@vconzola @michaelkro super nit, but wondering if this should be "VM Transformations Completed" OR "VM transformations completed" ?  this would affect both the Infotip title & the value in the table, if we decide to change it